apps/admin/frontend/src/api.ts

```diff
@@ -223,16 +223,20 @@ export const getPrintedBallots = {
 
 // Grouped Invalidations
 
-async function invalidateCastVoteRecordQueries(queryClient: QueryClient) {
-  await queryClient.invalidateQueries(getCastVoteRecordFileMode.queryKey());
-  await queryClient.invalidateQueries(getCastVoteRecordFiles.queryKey());
-  await queryClient.invalidateQueries(getCastVoteRecords.queryKey());
+function invalidateCastVoteRecordQueries(queryClient: QueryClient) {
+  return Promise.all([
+    queryClient.invalidateQueries(getCastVoteRecordFileMode.queryKey()),
+    queryClient.invalidateQueries(getCastVoteRecordFiles.queryKey()),
+    queryClient.invalidateQueries(getCastVoteRecords.queryKey()),
+  ]);
```

Comment:
I changed this when I was struggling with the queries in tests. Can revert it if it's confusing but doesn't matter one way or another functionally.